cpuset/OWNERS

```diff
@@ -0,0 +1,7 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - derekwaynecarr
```

Comment:
I suggest finding an additional approver?

Reply:
(You could add yourself if you wanted, moving the package qualifies you IMO)

Reply:
Done

Reply:
I'd nominate  @fromanirh here as well. You can add me too so we have bigger pool of people.

Reply:
I should be here as well

Reply:
can we add @ffromani, @dchen1107, and @SergeyKanzhelev here as well. Thank you!

Reply:
Done. Omissions were not thoughtful, I am just new here. Thanks for feedback.